README.md

```diff
@@ -3,120 +3,15 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE.md)
 
 <p float="left">
-  <img src="https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/neuropoly_logo.png" height="120" />
-  <img src="https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/mila_logo.png" height="120" /> 
-  <img src="https://github.com/neuropoly/ivado-medical-imaging/raw/master/images/ivado_logo.png" height="120" />
+  <img src="images/neuropoly_logo.png" height="80" />
+  <img src="images/mila_logo.png" height="80" />
+  <img src="images/ivado_logo.png" height="80" />
 </p>
 
 # IVADO Medical Imaging
 Comprehensive and open-source repository of deep learning methods for medical data segmentation.
 Collaboration between MILA and NeuroPoly for the IVADO project on medical imaging.
 
```

Comment:
Do we want to include a link to RTD here? Even if it's not final, people visiting our repo could have access to the doc.

Reply:
step-by-step: https://github.com/neuropoly/ivado-medical-imaging/issues/248